### DISCUSSION TITLE
starstruck

### DISCUSSION THREAD

**Discussion by bingyun123** [drknzz/GitHub-Achievements#20] [Q&A] [answered]
Hi all,

[repo](https://github.com/bingyun123/custom_chart_viz)

**Comment by eduardconstantin** [comment, marked as answer]
Hello! I have this [repo](https://github.com/eduardconstantin/Comic-book-button-anim) that is close for this achievement.

**Reply by eduardconstantin** [reply]
Thank you!

**Reply by eduardconstantin** [reply]
How did I get the achievement? I still have 12 stars on that repo

**Reply by eduardconstantin** [reply]
I see now, but that is not right. I was expecting to get stars from other users I needed just 4 more stars. Shouldn't the achievement disappear if the criteria is not met anymore?

**Reply by bingyun123** [reply]
yes,ok,I give you my star.

**Reply by eduardconstantin** [reply]
Thank you! Did you use different accounts to star the repo?

**Reply by spidybroz** [reply]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀 

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

#### [maiz-an](https://github.com/maiz-an)

**Comment by spidybroz** [comment]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀 

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

#### [maiz-an](https://github.com/maiz-an)

**Comment by nanisadw3** [comment]
Hi everyone! Doing **Star for Star (S4S)** to unlock achievements. 🌟

🔗 **Mis repositorios:**
- [nanisadw3/nanisadw3](https://github.com/nanisadw3/nanisadw3)
- [nanisadw3/mcp-banxico](https://github.com/nanisadw3/mcp-banxico)
- [nanisadw3/Gordito_AI](https://github.com/nanisadw3/Gordito_AI)

Drop your links below and I'll star back instantly! 🚀
